### ISSUE TITLE
Shim use in G2Z

### ISSUE THREAD

**Issue by apyarnold** [JaredC01/Galileo2#22]
Should a shim be used on pages 19 of the G2Z installation instructions the same as on page 32 of the G2E instructions?
Thanks

**Comment by JaredC01** [comment]
Yes, for the carrier against bearing, there should be a shim in both. The manual will need a full once-over prior to release, so that's on the list!
